src/meta/src/controller/streaming_job.rs

```diff
@@ -1605,6 +1609,112 @@ impl CatalogController {
             .await
     }
 
+    pub async fn update_sink_config_by_sink_id(
```

Comment:
It's unlikely to have multiple sink fragments in a sink, so no need to return a map of multiple fragments.

Besides, if we don't store actor ids in `Mutation::AlterSinkConfig`, we don't even need to return the actor ids here.